public/app/features/admin/UserOrgs.tsx

```diff
@@ -324,9 +324,11 @@ export class AddToOrgModal extends PureComponent<AddToOrgModalProps, AddToOrgMod
         if (this.state.pendingUserId) {
           await updateUserRoles(this.state.pendingRoles, this.state.pendingUserId!, this.state.pendingOrgId!);
           // clear pending state
-          this.state.pendingOrgId = null;
-          this.state.pendingRoles = [];
-          this.state.pendingUserId = null;
+          this.setState({
```

Comment:
@briangann can i check if this change is ok? in the original PR the state mutation was mentioned but the conversation was resolved and the PR merged... is there a reason for mutating `this.state` directly? if so, we can ignore the `eslint` rule, but i'd prefer not to for obvious reasons 😅

Reply:
i will check if there is a better alternative

Reply:
i think this change will be fine